lfg.go

```diff
@@ -0,0 +1,2 @@
+package main
```

Comment:
Is this for the lfg command?  There's a handful of folders with the commands already in them.